src/source/PeerConnection/Retransimitter.c

```diff
@@ -95,17 +91,19 @@ STATUS resendPacketOnNack(PRtcpPacket pRtcpPacket, PKvsPeerConnection pKvsPeerCo
         CHK(retStatus == STATUS_SUCCESS, retStatus);
 
         if (pRtpPacket != NULL) {
-            CHK_STATUS(constructRetransmitRtpPacketFromBytes(pRtpPacket->pRawPacket, pRtpPacket->rawPacketLength,
-                    pSenderTranceiver->sender.rtxSequenceNumber, pSenderTranceiver->sender.rtxPayloadType, pSenderTranceiver->sender.rtxSsrc, &pRtxRtpPacket));
-            pSenderTranceiver->sender.rtxSequenceNumber++;
+            if (pSenderTranceiver->sender.payloadType == pSenderTranceiver->sender.rtxPayloadType) {
```

Comment:
We should really start putting proper comments. Not part of this CR obviously, but I see no comments in major logic aspect while I see comments like // resendPacket which tells me nothing as a reader.